src/poliastro/bodies.py

```diff
@@ -5,9 +5,27 @@
 
 * Sun (☉)
 * Earth (♁)
+* Moon (☾)
+* Mercury (☿)
+* Venus (♀)
+* Mars (♂)
+* Jupiter (♃)
+* Saturn (♄)
+* Uranus (⛢)
+* Neptune (♆)
+* Pluto (♇)
```

Comment:
haha you are right! But, in [DE430] linked in discussions above, there is a table like this:
![imagen](https://user-images.githubusercontent.com/18404378/29027586-9630e31a-7b81-11e7-88bf-fe6de365d6f8.png)
so it appears that even for JPL, Pluto has a higher status than Ceres and other dwarf planets 😄